docs/src/list_of_supported_models.md

```diff
@@ -34,9 +37,11 @@ independent assessment.
 [EvoTrees.jl](https://github.com/Evovest/EvoTrees.jl) | - | EvoTreeRegressor, EvoTreeClassifier, EvoTreeCount, EvoTreeGaussian, EvoTreeMLE | medium | tree-based gradient boosting models
 [EvoLinear.jl](https://github.com/jeremiedb/EvoLinear.jl) | - | EvoLinearRegressor | medium | linear boosting models
 [GLM.jl](https://github.com/JuliaStats/GLM.jl) | [MLJGLMInterface.jl](https://github.com/JuliaAI/MLJGLMInterface.jl) | LinearRegressor, LinearBinaryClassifier, LinearCountRegressor | medium² |
+[Imbalance.jl](https://github.com/JuliaAI/Imbalance.jl) | - | RandomOversampler, SMOTENC, TomekUndersampler, ClusterUndersampler, SMOTE, SMOTEN, ROSE, RandomUndersampler, ENNUndersampler, BorderlineSMOTE1, RandomWalkOversampler | low | 
 [LIBSVM.jl](https://github.com/mpastell/LIBSVM.jl) | [MLJLIBSVMInterface.jl](https://github.com/JuliaAI/MLJLIBSVMInterface.jl) | LinearSVC, SVC, NuSVC, NuSVR, EpsilonSVR, OneClassSVM | high | also via ScikitLearn.jl
 [LightGBM.jl](https://github.com/IQVIA-ML/LightGBM.jl) | - | LGBMClassifier, LGBMRegressor | high | 
 [Flux.jl](https://github.com/FluxML/Flux.jl) | [MLJFlux.jl](https://github.com/FluxML/MLJFlux.jl) | NeuralNetworkRegressor, NeuralNetworkClassifier, MultitargetNeuralNetworkRegressor, ImageClassifier | low |
+[MLJBalancing.jl](https://github.com/JuliaAI/MLJBalancing.jl) | - | BaggedEnsembleClassifier | low | 
```

Comment:
Does this invite renaming the classifier in `MLJBalancing` from `BalancedBaggingClassifier` to `BaggedEnsembleClassifier`? I think the presence of the word "Balanced" is pivotal because the way this differs from ordinary bagging really is that each subset is guaranteed to be balanced.

Reply:
Good spot. That's a typo.

Reply:
fixed

Reply:
> Good spot. That's a typo.
That's okay. I do these day and night in `Imbalance.jl` :D.